src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const partner = `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+      {person.age && <p className="Person__age">I am {person.age}</p>}
+
+      {person.isMarried ? (
+        <p className="Person__partner">{partner}</p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
```suggestion
        <p className="Person__partner">
          {person.isMarried 
            ? `${person.partnerName} is my ${partner}`
            : I am not married
          }
        </p>
```
That was meant in the last comment - condition **inside** `p` tag

just FYI